src/aind_metadata_service/sharepoint/nsb2023/mapping.py

```diff
@@ -3071,6 +5603,11 @@ def has_hp_procedure(self) -> bool:
             NSBProcedure.INJECTION_FIBER_OPTIC_IMP,
             NSBProcedure.ISIGUIDED_INJECTION_WITH,
             NSBProcedure.STEREOTAXIC_INJECTION_WIT,
+            # Adding new names for supported HPs
```

Comment:
@saskiad Can you check whether the new HPs and Craniotomies look right? The options in the request form got re-named and some new ones got added so I added them here based on the naming (You can check models.NSBProcedure for the values)

Reply:
These are the "new" procedures being mapped to Headframe:
-  "Sx-11&19 HP Only"
- "Sx-11&19 HP Transcranial"
- "Sx-12 Stereotaxic Injection (with Headpost)"
- "Sx-21 Fiber Optic Implant (with Headpost)"
- "Sx-21 Injection + Fiber Optic Implant (with Headpost)"
- "Sx-26 ISI-guided Injection (with Headpost)"

These are the "new" procedures being mapped to Craniotomy:
- "Sx-1	Visual Ctx 2P"
-  "Sx-2	Frontal Ctx 2P"
- "Sx-3	Motor Ctx 2P"
- "Sx-4	WHC 2P"
- "Sx-5	INJ + Visual Ctx 2P"
- "Sx-6	Grid INJ (6 or 9) + Visual Ctx 2P"
- "Sx-8	INJ + Motor Ctx 2P"
- "Sx-9	Grid INJ (6 or 9) + Motor Ctx 2P"
- "Sx-14 Visual Ctx NP"
- "Sx-15 WHC NP"
- "Sx-16 INJ + WHC NP"

Reply:
This looks right.